### PR TITLE
Adds new add column workflow

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds the new workflow to create columns in a table:

new_column3

@xavijam: can you have a look to the code, please? Also, I'm going to need some help enabling the rename mode on the column name. Can you give me a hand?

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4240/))

File: app/assets/stylesheets/table/table.css.scss [hunk 1]
```diff
@@ -7,6 +7,20 @@
   @import "../table/table-sprite";
   @import "../old_elements/progress-bar";
 
+  @-webkit-keyframes highlight-fade {
```
Comment (xavijam) [inline app/assets/stylesheets/table/table.css.scss]: Not sure if we should add more CSS content to table file, could we move it to another file?

Comment (javierarce) [inline app/assets/stylesheets/table/table.css.scss]: Ok, sure.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4243/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4244/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4251/))

**Comment by javierarce** [comment]
@xavijam: can you review this? Regarding the setTimeouts, I tried to come up with a solution to avoid them but I'm afraid we'll need them.

**Comment by javierarce** [comment]
New workflow: 

new_column3

File: lib/assets/javascripts/cartodb/table/header_view.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ var HeaderView = cdb.admin.HeaderView = cdb.core.View.extend({
 
     if (HeaderView.colOptions === undefined) {
       HeaderView.colOptions= new cdb.admin.HeaderDropdown({
+        user: this.options.user,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/header_view.js]: Why do you need user now?

File: lib/assets/javascripts/cartodb/table/actions_menu.js [hunk 3]
```diff
@@ -150,6 +150,7 @@
         v.bind('delete',    self._openDeleteLayerDialog, self);
         v.bind('show',      self.show, self);
         v.bind('destroy',   self._removeLayer, self);
+        v.bind('addColumn', self._addColumn, self);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/actions_menu.js]: Why do we need addColumn within actions-menu class?

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/actions_menu.js]: Ah, ok, coming from layer panel view, :+1:.

File: lib/assets/javascripts/cartodb/table/header_view.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/header_view.js]: To use the flag for the new modals.

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/header_view.js]: :+1:

**Comment by xavijam** [comment]
What happened with the 2 renders problem?

**Comment by saleiva** [comment]
\o/

**Comment by javierarce** [comment]
Seems to be working as expected. The `addColumn` method is only called once, so probably I was mistaken.

**Comment by xavijam** [comment]
Ok, apart from the ```setTimeout``` thing you mentioned, for me it is ok. If you don't find a better solution, good to go, but probably worth to ask rest of the @CartoDB/frontend.

**Comment by saleiva** [comment]
I guess you guys are generating that number as a random. What about doing something like column_XXXXX? without the column before it looks like an error to me

**Comment by xavijam** [comment]
It is a timestamp, I've done something similar in https://github.com/CartoDB/cartodb/pull/3896.

**Comment by javierarce** [comment]
Ok, I'll use column_TIMESTAMP.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4278/))

**Pull request merged**